tests/foreman/api/test_computeresource_vmware.py

```diff
@@ -124,3 +125,60 @@ def test_positive_provision_end_to_end(
         delay=10,
     )
     assert host.read().build_status_label == 'Installed'
+
+
+@pytest.mark.e2e
```

Comment:
we already cover normal discovery in e2e, so I think we can exclude this test for e2e

Reply:
updated